src/features/pages/options/accumulator/index.tsx

```diff
@@ -1,42 +1,146 @@
 import React from 'react'
-import {
-    accumulatorFAQ,
-    accumulatorMarkets,
-    accumulatorOptionsContentItems,
-    accumulatorPlatforms,
-} from './data'
-import { Localize } from 'components/localization'
-import Typography from 'features/components/atoms/typography'
+import { Features, AccordionBlock } from '@deriv-com/blocks'
+import { Text } from '@deriv/quill-design'
+import * as styles from './accumulators_faq.module.scss'
+import { accumulatorCards } from './data'
+import { Localize, localize } from 'components/localization'
 import OptionsLayout from 'features/components/templates/options-layout'
-import OptionsContent from 'features/components/templates/options-content'
-import AvailableMarketPlatforms from 'features/components/organisms/available-markets-platforms'
-import OptionsFaq from 'features/components/templates/options-faq'
-import Flex from 'features/components/atoms/flex-box'
 
 const AccumulatorsOptions = () => {
     return (
         <>
             <OptionsLayout
-                heading="_t_What are accumulator options?_t_"
-                description="_t_Accumulator options allow you to predict whether the current spot price remains within a predefined range from the previous spot price._t_"
+                heading="_t_Move fast,<br/>when the market doesn’t_t_"
+                description="_t_Amplify your gains in sideways markets with accumulator options.<br/> Up to 5% compounded growth per tick._t_"
+                has_content_block
             >
-                <Typography.Heading
-                    as="h2"
-                    align="center"
-                    padding_block="6x"
-                    md={{ padding_block: '20x' }}
-                    size="small"
-                >
-                    <Localize translate_text="_t_How do accumulator options work?_t_" />
-                </Typography.Heading>
-                <OptionsContent items={accumulatorOptionsContentItems} />
-                <Flex.Box justify="center" align="center" direction="col">
-                    <AvailableMarketPlatforms
-                        markets={accumulatorMarkets}
-                        platforms={accumulatorPlatforms}
+                <Features.Card
+                    title={
+                        <Localize translate_text="_t_Why trade accumulator options on Deriv_t_" />
+                    }
+                    description={
+                        <Localize translate_text="Looking for a fast-paced, strategic and potentially explosive way to trade?" />
+                    }
+                    cols="three"
+                    cards={accumulatorCards}
+                />
+                <div id="faqs">
```

Comment:
Why this `div` and `id` needed?

Reply:
Needed in order to scroll to the specific section when clicking the "Read FAQs" button. If I put the id in the AccordionBlock it doesn't scroll to the section